CONTRIBUTING.md

```diff
@@ -140,6 +140,15 @@ Once you've discussed your proposed feature/fix/etc. with a team member, and you
 1. Create & push a feature branch
 1. Create a [Draft Pull Request (PR)](https://github.blog/2019-02-14-introducing-draft-pull-requests/)
 1. Work on your changes
+1. Build and see if it works. Consult [How to build OpenConsole](./doc/building.md) if you have problems.
+
+### Testing
+
+Testing is a key component in the development workflow. Both Windows Terminal and Windows Console use TAEF(the Test Authoring and Execution Framework) as the main framework for testing.
+
+If your changes affect existing test cases, or you're working on brand new features and also the accompanying test cases, see [TAEF](./doc/TAEF.md) for more information about how to validate your work locally.
+
+> Note that you can not run TAEF tests through Visual Studio. There may be plugins that can help you with that, but this approach is not recommended or supported.

```

Comment:
_technically_, there is a way to run the tests in VS that @DHowett uses. I think it's more work than doing it from the commandline. I think it's fine to leave this as-is, just FYI

Reply:
The `Taef.Redist.Wlk` NuGet package has an MSTest/TAEF adapter inside of it that will let you browse and execute TAEF tests inside of Visual Studio. The performance of this mechanism leaves a lot to be desired and it is finnicky with the "debug this test" and other features you might expect when you see them light up inside Visual Studio. So it's probably best to just avoid it and do it the reliable way with `TE.exe` itself.

Reply:
FYI, when I need to debug tests in Visual Studio I set the debugging command on the test project to `$(OutDir)TE.EXE`, and the command arguments to `/inproc $(TargetPath)` (potentially with `/name` for a specific test case). I find that really useful if I need to step through the code. I'm not sure it works for all of the unit test projects, though.

Reply:
@j4james That is a good approach I've also been using myself. But I think adding it here sounds like a overkill. It shouldn't be too hard for people to find the `/inproc` thing in TAEF documentation.